hail/python/hail/vds/combiner/variant_dataset_combiner.py

```diff
@@ -701,6 +709,16 @@ def maybe_load_from_saved_path(save_path: str) -> Optional[VariantDatasetCombine
                     "entry types. Overwriting with types from supplied VDS.")
         gvcf_reference_entry_fields_to_keep = ref_entry_tmp
 
+        all_entry_types = chain(vds.reference_data._type.entry.items(),
+                                vds.variant_data._type.entry.items())
+        call_fields_tmp = {name for name, typ in all_entry_types if typ == hl.tcall} - {'LGT', 'GT'}
```

Comment:
These are the call fields from the vds, right? Let's use a descriptive name. Maybe `vds_call_fields`?

Reply:
👍